### PR TITLE
Support millisecond extraction from time/timestamp values

### LINKED ISSUE #9524: Support extracting milliseconds from a timestamp
```
presto> select extract(second from (timestamp '2017-10-24 23:00:23.206'));
 _col0
-------
    23
```

but
```
presto> select extract(millisecond from (timestamp '2017-10-24 23:00:23.206'));
Query 20171208_215149_00048_uhmtc failed: line 1:8: Invalid EXTRACT field: millisecond
```

IMO this should be supported.

### PR DESCRIPTION
**Pull request by kasiafi**
Fixes #9524

### CODE REVIEW

**Comment by findepi** [comment]
This looks good overall. Certainly documentation needs update to include the new function.
However, please wait for @martint 's opinion first.

**Comment by kasiafi** [comment]
I removed `EXTRACT` changes.

**Comment by kasiafi** [comment]
@martint I applied your comments. Could you please review?

File: presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctionsBase.java [hunk 1]
```diff
@@ -265,6 +265,7 @@ public void testTimeZone()
     @Test
     public void testPartFunctions()
     {
+        assertFunction("millisecond(" + TIMESTAMP_LITERAL + ")", BIGINT, (long) TIMESTAMP.getMillisOfSecond());
```
Comment (martint) [inline presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctionsBase.java]: There should be a test for each of the variants added above (interval, time w/ timezone, time, timestamp w/ timezone)

**Comment by kasiafi** [comment]
@martint I added the tests that were missing. Could you please review?

**Pull request merged**